mlos_bench/mlos_bench/tests/services/local/local_exec_test.py

```diff
@@ -59,8 +62,8 @@ def test_resolve_script(local_exec_service: LocalExecService) -> None:
     """
     script = "os/linux/runtime/scripts/local/generate_kernel_config_script.py"
     script_abspath = local_exec_service.config_loader_service.resolve_path(script)
-    orig_cmdline = f". env.sh && {script}"
-    expected_cmdline = f". env.sh && {script_abspath}"
+    orig_cmdline = f". env.sh && {script} --input foo"
+    expected_cmdline = f". env.sh && {script_abspath} --input foo"
```

Comment:
@motus this is unrelated to this change, but adds additional testing to the case you asked about here:
https://github.com/microsoft/MLOS/pull/461/files#r1349384116